{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_forms.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+from django.utils.translation import ugettext_lazy as _
```

Comment:
```suggestion
from django.utils.translation import gettext_lazy as _
```
`ugettext_lazy` is deprecated right?